nmrfx-core/src/main/java/org/nmrfx/datasets/DatasetBase.java

```diff
@@ -18,6 +19,7 @@ public class DatasetBase {
     public final static int UCSF_HEADER_SIZE = 180;
     public final static int LABEL_MAX_BYTES = 16;
     public final static int SOLVENT_MAX_BYTES = 24;
+    public final static String DATASET_PROJECTION_TAG = "_proj_";
```

Comment:
This probably isn't the best spot for it since Datasets don't track if they are projections, but since I needed to access it from nmrfx-core, I couldn't put it in DatasetAttributes